modin/conftest.py

```diff
@@ -693,3 +693,18 @@ def s3_resource(s3_base):
         if not cli.list_buckets()["Buckets"]:
             break
         time.sleep(0.1)
+
+
+@pytest.fixture
```

Comment:
this was moved from `test_internals.py` in order to use this fixture in `test_groupby.py`